src/bp-templates/bp-nouveau/buddypress/moderation/moderation-reported-content-entry.php

```diff
@@ -39,7 +39,7 @@
     </td>
     <td class="moderation-content-excerpt">
 		<?php
-		$content_excerpt = bp_moderation_get_content_excerpt( bp_get_moderation_item_id(), bp_get_moderation_item_type() );
+		$content_excerpt = bp_moderation_get_content_excerpt( bp_get_moderation_item_id(), bp_get_moderation_item_type(), false );
```

Comment:
@janakkaneriya007 As we adding default value for 3rd argument as `false` so no need to pass `false`